playwright/config/.env.at23

```
@@ -1,26 +1,4 @@
 ENV_NAME="at23"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)